### ISSUE TITLE
This plugin causes issues on stackoverflow comment system

### ISSUE THREAD

**Issue by elieobeid7** [slgobinath/uget-chrome-wrapper#48]
I have firefox quantum, this issue is causing issues with stackoverflow comment system detailed here

https://meta.stackexchange.com/questions/305447/every-time-i-comment-i-have-to-refresh-my-browser-to-see-it

**Comment by slgobinath** [comment]
Thanks for pointing this out.
I will have a look at this.

**Comment by slgobinath** [comment]
Hi,
Currently, I am looking at this issue but for testing purposes, I need to reproduce this issue multiple times.
Is there a way to test this behavior without spamming StackOverflow community?

**Comment by elieobeid7** [comment]
Do the tests here

https://meta.stackexchange.com/questions/3122/formatting-sandbox

That thread is meant for testing

**Comment by slgobinath** [comment]
Fixed with https://github.com/slgobinath/uget-chrome-wrapper/commit/aaec89593a581b8c2f0c9cb6b77c34e13891e2a2. Will be included in the next release.

**Comment by slgobinath** [comment]
New addon is released. Please check whether the problem has been resolved.

Thanks

**Comment by elieobeid7** [comment]
The problem on stackexchange is fixed, however:

1. The plugin stopped working after the update, try to download https://vivaldi.com/ or anything using the new plugin on firefox quantum on linux. When you press download nothing happens, the plugin isn't working and it's also breaking firefox's download system

2.  The plugin isn't getting updated, I had to remove it from firefox and reinstall it from http://addons.mozilla.org

**Comment by slgobinath** [comment]
Hi,

> The plugin stopped working after the update

Did you upgrade `uget-chrome-wrapper`? I am able to download Vivaldi using uGet.

> The plugin isn't getting updated

I will have a look at this

**Comment by elieobeid7** [comment]
Ok so now it works

- I had to redownload and reinstall `uget-chrome-wrapper` from the site

- Remove the extension and reinstall it from addons.mozilla.org

The process should be automated though because most people won't even know that this issue exists on github. But in any case, everything works now. Thank you so much.

**Comment by slgobinath** [comment]
Hi,
Thanks for the confirmation.
If you have installed `uget-chrome-wrapper` using the shell script, there is no way I can provide automatic update. If Windows, sorry I don't have a Windows machine to compile the installer. I will release it soon.

> Remove the extension and reinstall it from addons.mozilla.org

I also experienced that issue but I guess I have to check with Firefox side too. Once I resolved it, I can ask in the extension to upgrade the wrapper.
